sykmeldingalerts-dev.yaml

```diff
@@ -10,28 +10,28 @@ spec:
       channel: '#sykmelding-alerts-dev'
   alerts:
     - alert: syfosm-app-nede
-      expr: kube_deployment_status_replicas_available{deployment=~"syfosmregler|syfosmregister|syfotekster|syfohelsenettproxy|syfosmpapirregler|syfosmmanuell-backend|syfosmmanuell"} == 0
+      expr: kube_deployment_status_replicas_available{deployment=~"syfosmregler|syfosmregister|syfotekster|syfohelsenettproxy|syfosmpapirregler|syfosmmanuell-backend|syfosmmanuell|egenmeldt-sykmelding-backend"} == 0
       for: 2m
       description: "{{ $labels.deployment }} er nede"
       action: "Se `kubectl kubectl get pod  | grep {{ $labels.deployment }}` for å se podder, og `kubectl logs $podnavn` for logger"
       sla: respond within 1h, during office hours
       severity: danger
     - alert: syfosm-asynkron-app-nede
-      expr: kube_deployment_status_replicas_available{deployment=~"syfosmoppgave|syfosmarena|syfosmvarsel|syfosminfotrygd|syfosmsak|syfosmmottak|syfosmapprec|syfosmpapirmottak|syfovarsel|pale-2|pale-2-sak|helse-sykmelding-router|pale-2-register"} == 0
+      expr: kube_deployment_status_replicas_available{deployment=~"syfosmoppgave|syfosmarena|syfosmvarsel|syfosminfotrygd|syfosmsak|syfosmmottak|syfosmapprec|syfosmpapirmottak|syfovarsel|pale-2|pale-2-sak|helse-sykmelding-router|pale-2-register|egenmeldt-sykmelding-backend"} == 0
```

Comment:
Denne dekkes av alerten over :)

Reply:
Den over er for apper som andre har synkrone koblinger til, denne er for apper som kun er asynkrone, der vi tåler mer nedetid ;)

Reply:
ifix